specification/containerregistry/data-plane/Microsoft.ContainerRegistry/preview/2019-08-15/containerregistry.json

```diff
@@ -1672,7 +1673,7 @@
           }
         },
         "changeableAttributes": {
-          "$ref": "#/definitions/ChangeableAttributes",
+          "$ref": "#/definitions/ManifestChangeableAttributes",
```

Comment:
Moving to this type will add the private quarantine fields to the public API.  Is that what you'd like to have happen?

Reply:
It is already returned as a payload, and is also supported by PATCH as I see it in the service code

Reply:
Would you please add:
`"x-accessibility": "internal",`
to both `quarantineState` and `quarantineDetails`?

Reply:
done